diff --git a/colorer/src/Colorer-library/src/colorer/io/FileInputSource.cpp b/colorer/src/Colorer-library/src/colorer/io/FileInputSource.cpp
index 2c0987f..82fd919 100644
--- a/colorer/src/Colorer-library/src/colorer/io/FileInputSource.cpp
+++ b/colorer/src/Colorer-library/src/colorer/io/FileInputSource.cpp
@@ -1,7 +1,6 @@
 #include<stdio.h>
 #include<string.h>
 #include<sys/stat.h>
-#include<sys/timeb.h>
 #include<fcntl.h>
 #include<time.h>
 
