diff --git a/multiarc/src/formats/rar/unrar/os.hpp b/multiarc/src/formats/rar/unrar/os.hpp
index bb9eb0b0..079d358b 100644
--- a/multiarc/src/formats/rar/unrar/os.hpp
+++ b/multiarc/src/formats/rar/unrar/os.hpp
@@ -175,7 +175,7 @@
 #define SAVE_LINKS
 #endif
 
-#if defined(__linux) || defined(__FreeBSD__)
+#if (defined(__linux) && !defined(__ANDROID__)) || defined(__FreeBSD__)
 #include <sys/time.h>
 #define USE_LUTIMES
 #endif
